packages/angular-cli/package.json

```diff
@@ -25,6 +25,7 @@
   "homepage": "https://github.com/angular/angular-cli",
   "dependencies": {
     "@angular-cli/ast-tools": "^1.0.0",
+    "@angular-cli/base-href-webpack": "^1.0.0",
```

Comment:
@hansl this needs to be in the root package.json file as well. it's causing errors that the library isn't found when running `ng` commands after a fresh install of angular-cli